linera-service/tests/fixtures/evm_example_execute_message.sol

```diff
@@ -8,6 +8,10 @@ contract ExampleExecuteMessage {
     uint64 value;
     Linera.MessageId last_message_id;
 
+    constructor(uint64 test_value) {
+       require(test_value == 42);
```

Comment:
Can you modify the constructor so that it takes the `test_value` and sets one of its field to it and later check, by calling contract's method, whether it was set properly?

It's not obvious that in this test the constructor is even called.

Reply:
Well, you can run with 41 and see that the test does not pass.